src/core/annotation.js

```diff
@@ -661,6 +677,63 @@ class WidgetAnnotation extends Annotation {
     return !!(this.data.fieldFlags & flag);
   }
 
+  /**
+   * Parse widget annotation appearance and extract font information.
+   *
+   * @public
+   * @memberof WidgetAnnotation
+   * @param {number} flag - Hexadecimal representation for an annotation
+   *                        field characteristic
+   * @return {Promise}
+   */
+  parseAppearance(params) {
+    if (!params.pdfManager || !params.pdfManager.pdfDocument) {
+      return Promise.resolve(this);
+    }
+
+    let data = this.data;
+    let self = this;
+
+    let opList = new OperatorList();
+    let appearanceStream = new Stream(stringToBytes(data.defaultAppearance));
+    let formFonts = params.pdfManager.pdfDocument.acroForm.get('DR');
+    opList.acroForm = params.pdfManager.pdfDocument.acroForm;
```

Comment:
We should not put additional member variables on the operator list. I don't really see why it should be necessary and it breaks the expected structure of the object.

Reply:
Here I think I need your recommendation, I just not familiar enough with all details of the PDF.JS code. Is it fine to extend OperatorList object with new property acroForm? In this case I tried to keep changes local in order before calling evaluator...